package.json

```diff
@@ -51,6 +52,7 @@
     "react-scripts": "^3.3.0",
     "react-select": "^3.0.8",
     "redux-oidc": "^3.1.5",
+    "scroll-to-element": "^2.0.3",
```

Comment:
the first thing that comes to my mind when I see this dependency is that it looks a lot like [`Element.scrollIntoView()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView). Is this dep used for polyfilling?